skiko/build.gradle.kts

```diff
@@ -427,6 +428,32 @@ kotlin {
     }
 }
 
+fun configureCinterop(
```

Comment:
Is the property `enableCInteropCommonization` for something else? (docs doesn't say its purpose clearly)